elastic/kibana
# [Watcher] Improve usability of Simulate flyout

Closes https://github.com/elastic/kibana/issues/153327

This PR adds a "Status" column to the table in the Simulate flyout in Watcher and adds an element on top which shows whether the condition is met. When the condition is met, the Status column shows the `status` value of each action from the  `result.actions` field in the Json below the table. When the conditon is not met, the Status column should display "not simulated"/"not executed"/"throttled" depending on the mode of the action (see screenshots below).

<img width="782" alt="Screenshot 2023-03-30 at 15 16 25" src="https://user-images.githubusercontent.com/59341489/228868052-c48df0f7-3d5e-472e-9029-086886bb6902.png">
<img width="782" alt="Screenshot 2023-03-30 at 15 17 08" src="https://user-images.githubusercontent.com/59341489/228868068-0f2dabc5-ad8d-4bd6-b5b5-eb20e08aa125.png">

**How to test:**
1. Add some sample data set so that you have a certain number of hits.
2. Go to Watcher -> Create advanced watch.
3. In the Json editor, modify the condition so that it is or is not met.
4. Click on the Simulate tab and select some mode for each action.
5. Click "Simulate watch" and verify that the flyout correctly shows if the condition is met or not and check if the Status column contains the correct value.
6. Repeat the steps above with different action mode values and for both cases when the condition is and is not met.

https://user-images.githubusercontent.com/59341489/228868091-2e3d7775-efea-4bea-9257-a2a4ec00436e.mov



### Checklist

Delete any items that are not applicable to this PR.
